tests/core/config/checkers/test_scenario_config_checker.py

```diff
@@ -293,6 +293,19 @@ def test_check_tasks_in_sequences_exist_in_scenario_tasks(self, caplog):
         assert len(Config._collector.warnings) == 0
         assert len(Config._collector.infos) == 0
 
+        config._sections[ScenarioConfig.name]["new"].add_sequences({"sequence_2": ["task_config"]})
```

Comment:
```suggestion
        config._sections[ScenarioConfig.name]["new"].add_sequences({"sequence_2": [task_config_1, "task_config"]})
```
I propose testing a list consisting of both TaskConfig and non-TaskConfig

Reply:
haha sure why not